### DISCUSSION TITLE
Version 1.22.1

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#3822] [Changelog]
## Bugfixes
- [#3789](https://github.com/ScratchAddons/ScratchAddons/issues/3789) editor-comment-previews stops working after leaving and reentering editor. (prio-3)
- [#3789](https://github.com/ScratchAddons/ScratchAddons/issues/3789) editor-comment-previews shows previews for comments with only whitespaces. (prio-4)
- [#3789](https://github.com/ScratchAddons/ScratchAddons/issues/3789) editor-comment-previews does not hide preview on right click. (prio-3)
- [#3789](https://github.com/ScratchAddons/ScratchAddons/issues/3789) editor-comment-previews has reduced readability on Firefox. (prio-3)
- [#3789](https://github.com/ScratchAddons/ScratchAddons/issues/3789) editor-comment-previews does not work when hovering over text input. (prio-4)
- [#3793](https://github.com/ScratchAddons/ScratchAddons/issues/3793) search-my-stuff causes input to be unusable on Firefox. (prio-3)
- [#3793](https://github.com/ScratchAddons/ScratchAddons/issues/3793) search-my-stuff prevents scrolling using keyboard. (prio-3)
- [#3793](https://github.com/ScratchAddons/ScratchAddons/issues/3793) search-my-stuff logs error when switching tabs. (prio-4)
- [#3793](https://github.com/ScratchAddons/ScratchAddons/issues/3793) search-my-stuff does not allow Cmd key to be used in place of Ctrl. (prio-3)
- [#3794](https://github.com/ScratchAddons/ScratchAddons/issues/3794) Searching for new addons causes the items to flicker. (prio-3)
- [#3796](https://github.com/ScratchAddons/ScratchAddons/issues/3796) fullscreen does not support ask and wait prompt. (prio-3)
- [#3796](https://github.com/ScratchAddons/ScratchAddons/issues/3796) fullscreen does not support horizontally small screens. (prio-4)
- [#3798](https://github.com/ScratchAddons/ScratchAddons/issues/3798) hide-flyout shows the flyout when dragging the workspace. (prio-3)
- [#3807](https://github.com/ScratchAddons/ScratchAddons/issues/3807) editor-searchable-dropdowns has wrong border style. (prio-4)

**Comment by Samq64** [comment]
#3776 animated-thumb was still an editor add-on. (prio-4?)
